frontend/packages/console-dynamic-plugin-sdk/src/webpack/ConsoleRemotePlugin.ts

```diff
@@ -0,0 +1,84 @@
+import * as webpack from 'webpack';
+import * as readPkg from 'read-pkg';
+import * as semver from 'semver';
+import * as _ from 'lodash';
+import { ConsoleAssetPlugin } from './ConsoleAssetPlugin';
+import { ConsolePackageJSON } from '../schema/plugin-package';
+import { sharedVendorModules } from '../shared-modules';
+import { SchemaValidator } from '../validation/SchemaValidator';
+import { remoteEntryFile } from '../constants';
+import consolePkgMetadataSchema from '../../dist/schema/plugin-package';
+
+// eslint-disable-next-line
+const { ReplaceSource } = require('webpack-sources');
```

Comment:
Why a require statement and not an import?

Reply:
Because static import failed for some reason, if I recall correctly. Will check again.